WhereInTheWorld/WhereInTheWorld/DataManager.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DataManager.swift
+//  WhereInTheWorld
+//
+//  Created by Max Zvyagin on 2/13/20.
+//  Copyright © 2020 Max Zvyagin. All rights reserved.
+//
+
+import Foundation
+import MapKit
+
+public class DataManager{
+    
+    // static stuff
+    public static let sharedInstance = DataManager()
+    // this prevents others from using the default init
+    fileprivate init() {}
+    
+    // initialize a global array to keep track of favorites
+    var favorites = [String]()
+    
+    func loadAnnotationFromPlist(filename:String)->[Place]{
+       // source: https://learnappmaking.com/plist-property-list-swift-how-to/
+       var retArray = [Place]()
+       var data:NSDictionary? = nil
+       let path = Bundle.main.path(forResource: filename, ofType: "plist")!
+       let xml = FileManager.default.contents(atPath: path)
+       do{
+           data = try!  PropertyListSerialization.propertyList(from:xml!, options:.mutableContainersAndLeaves, format: nil) as! [String:AnyObject] as NSDictionary
+       }
+       let places: NSArray = data?.value(forKey: "places") as! NSArray
+       for x in places{
```

Comment:
```suggestion
       for x in places {
```

Use consistent syntax regarding curly brace usage throughout your code.